docs/directory-structure.md

```diff
@@ -131,8 +131,8 @@ Subdirectory for where all the images and figures refereneced in `README.md` liv
 
 ### Language Subdirectories
 Each example subdirectory could contain examples in multiple languages.
-Each language lives in a separate subdirectory.  Even if there is only one language, it should
-still be in its own subdirectory.
+Each language lives in a separate subdirectory.  If there is only one language, or if the example
+uses multiple languages, there is no need to create language subdirectories.
```

Comment:
I understand from context what you mean here, but I'm not sure the difference between "example *in* multiple languages" and "example *uses* multiple languages" will be generally understood. I think something like:

>Where possible, we appreciate having examples translated into several programming languages to help out all kinds of users. If you do so, add the code for each translation in a subdirectory of your main project named for the implementation language.

Might make it more clear what we expect.

Reply:
Thanks for the suggestions - I adopted your suggestion but changed it to third person.